files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -97,23 +97,25 @@ harden-flatpak:
 toggle-cups:
     #!/usr/bin/pkexec /usr/bin/bash
     if systemctl is-enabled --quiet cups; then
-      firewall-cmd --permanent --remove-port=631/tcp
-      firewall-cmd --permanent --remove-port=631/udp 
```

Comment:
Something is off here. The initial line should make the whole thing run as root:


```
#!/usr/bin/pkexec /usr/bin/bash
```

This is working for other commands. What behavior are you seeing for this one?

Reply:
I overlooked that line. The privilege escalation methods are all different, making this a bit complicated.